.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature Request
+about: Suggest an idea for this project
+title: "[FEATURE REQUEST]"
+labels: enhancement
+assignees: ''
+
+---
+
+## Is your feature request related to a problem? Please describe.
```

Comment:
If so, please give a short summary of the problem and how the feature would resolve the problem.